common/src/stack/command/stack/commands/disable/pallet/__init__.py

```diff
@@ -89,8 +89,8 @@ def run(self, params, args):
 			)
 
 		# Regenerate stacki.repo
-		os.system("""
+		stack.util._exec("""
```

Comment:
Think this is going to break because you are importing `stack.util`. The `_exec` function is added to the Command baseclass, so you can just do:
`self._exec`